frontend/Container.js

```diff
@@ -47,6 +46,7 @@ class Container extends React.Component {
     extraTabs: {[key: string]: () => React$Element},
   };
   state: State;
+  resizeTimeout: number = null;
```

Comment:
If it's a number it can't be null. `number | null` or `?number` would work though.